Update about.md (#3324)
SumoLogic/sumologic-documentation: docs/dashboards/about.md

* A Dashboard can have up to 100 queries.
* Dashboard chart properties are not retained when viewed from the Search page.
* Chart properties are not retained when a chart is added to a Dashboard from the Search page.

> * Dashboard queries cannot return more than 1440 data points. If the dashboard panel is wider than 1440 pixels, additional resolution in data will be down-sampled in order to render on the screen.

Reviewer: ```suggestion
* Dashboard queries cannot return more than 1440 data points.
```

> * No dashboard query can return more than 1440 data points. If the dashboard panel has more than 1440 pixels in width, additional resolution in data will be down-sampled to get rendered on the screen.

Reviewer: ```suggestion
* Dashboard queries cannot return more than 1440 data points. If the dashboard panel is wider than 1440 pixels, additional resolution in data will be down-sampled in order to render on the screen.
```

Reviewer 2: @JV0812 please change to `Dashboard queries cannot return more than 1440 data points.` The rest of the explanation is not necessary.

Reviewer: @oshoghi , all set! Can you pls approve the PR?
